packages/grid/src/vaadin-grid-dynamic-columns-mixin.js

```diff
@@ -90,6 +90,13 @@ export const DynamicColumnsMixin = (superClass) =>
       return columnTree;
     }
 
+    /** @protected */
+    _debounceUpdateColumnTree() {
+      this.__debouncerUpdateColumnTree = Debouncer.debounce(this.__debouncerUpdateColumnTree, microTask, () =>
```

Comment:
**nit:**

```suggestion
      this.__updateColumnTreeDebouncer = Debouncer.debounce(this.__updateColumnTreeDebouncer, microTask, () =>
```

Reply:
**unrelated note:** We could consider rewriting `microTask` to use `queueMicrotask` instead of a workaround based on `MutationObserver`. I believe using `MutationObserver` isn't efficient as it causes unnecessary changes in the DOM which subsequently cause the browser to repaint and so on.